### PR TITLE
Deprecated Capabilities fixed

### PR DESCRIPTION
**Pull request by kbayraktar**

### CODE REVIEW

File: src/main/java/org/testeditor/fixture/web/WebDriverFixture.java [hunk 1]
```diff
@@ -279,10 +280,8 @@ public WebDriver startFireFoxPortable(String browserPath) {
     private void launchChrome() {
         setupDrivermanager(ChromeDriverManager.getInstance());
         String browserName = BrowserType.CHROME;
-        DesiredCapabilities desiredCapabilities = DesiredCapabilities.chrome();
-        Object chromeCapability = populateBrowserSettings(desiredCapabilities, browserName);
-        desiredCapabilities.setCapability(ChromeOptions.CAPABILITY, chromeCapability);
-        driver = new ChromeDriver(desiredCapabilities);
+        ChromeOptions chromeOptions = (ChromeOptions) populateBrowserSettings(browserName);
```
Comment (jjelschen) [inline src/main/java/org/testeditor/fixture/web/WebDriverFixture.java]: As far as I can tell, `populateBrowserSettings` is only ever called when the browser name / type is statically known. So, you could break up the method into three separate ones (and probably a fourth one for the common parts). That way, you'd get rid of the switch/case, and you'd get type safety, i.e. you can loose the explicit type cast.

File: src/main/java/org/testeditor/fixture/web/WebDriverFixture.java [hunk 2]
```diff
@@ -358,9 +354,16 @@ private void populateChromeOption(List<BrowserSetting> options, ChromeOptions ch
                 chromeOptions.addArguments((String) option.getValue());
             });
         }
-
     }
 
+    private void populateIeOptions(List<BrowserSetting> options, InternetExplorerOptions ieOptions) {
+        if (options != null && !options.isEmpty()) {
```
Comment (jjelschen) [inline src/main/java/org/testeditor/fixture/web/WebDriverFixture.java]: `!options.isEmpty()` is not needed here

File: src/main/java/org/testeditor/fixture/web/WebDriverFixture.java [hunk 3]
```diff
@@ -308,27 +305,19 @@ private void launchFirefox() {
...
-        List<BrowserSetting> capabilities = new ArrayList<>();
-        List<BrowserSetting> options = new ArrayList<>();
-        populateWithBrowserSpecificSettings(BrowserType.IE, capabilities, options);
-        populateWithAdditionalCapabilities(desiredCapabilities, capabilities);
-    }
-
-    private Object populateBrowserSettings(DesiredCapabilities desiredCapabilities, String browserName) {
-        List<BrowserSetting> capabilities = new ArrayList<>();
+    private MutableCapabilities populateBrowserSettings(String browserName) {
+        //List<BrowserSetting> capabilities = new ArrayList<>();
```
Comment (jjelschen) [inline src/main/java/org/testeditor/fixture/web/WebDriverFixture.java]: … can be removed, right?

Comment (kbayraktar) [reply src/main/java/org/testeditor/fixture/web/WebDriverFixture.java]: Jepp :+1:

**Pull request merged**